bazel/deps.bzl

```diff
@@ -78,13 +78,6 @@ def stratum_deps():
             sha256 = P4RUNTIME_SHA,
         )
 
-    if "build_stack_rules_proto" not in native.existing_rules():
-        remote_workspace(
-            name = "build_stack_rules_proto",
-            remote = "https://github.com/stackb/rules_proto",
-            commit = "2f4e4f62a3d7a43654d69533faa0652e1c4f5082",
-        )
-
```

Comment:
This dep is needed by `rules_python`.
The only remaining Python scripts are https://github.com/stratum/stratum/blob/deddede3f8c5c643c21f8ece5f3c453e3ec659fd/stratum/hal/bin/dummy/BUILD#L15-L32

I'll see if a C++ rewrite or deletion is feasible.